tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -47,27 +47,13 @@ error: Unknown symbol \\"this\\"
   |     ^^^^ Unknown symbol \\"this\\"
 
 
-error: Variable this.instanceField is not reassignable 
```

Comment:
what happened here?

Reply:
`this` in a static method is an unknown symbol (in a best case scenario we can be more specific and say something like "this" isn't available in a static method). So it was a bug in the previous code where after we already know `this` is unknown we still check if it's reassignable. In the past we couldn't distinguish between `any` and an unknown variable, so we needed to account for the case that an `any` var isn't reassignable (even if it was actually an unknown symbol), today we can ignore these cases an reduce this redundant error message..